pkg/trace/api/normalizer.go

```diff
@@ -158,6 +158,9 @@ func normalize(ts *info.TagStats, s *pb.Span) error {
 			delete(s.Meta, "http.status_code")
 		}
 	}
+	if v, ok := s.Meta["version"]; ok {
+		s.Meta["version"] = normalizeTag(v)
```

Comment:
As you've noticed, this doesn't work for tags starting with numeric values. Note `normalizeTag` is a function which normalizes "_metrics tags_", not APM tags. It is usually applied to tags that end up in the metrics infrastructure.

Reply:
:+1: Will strike this in the next commit.